felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/KafkaKey.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.helsearbeidsgiver.felles.rapidsrivers
+
+import kotlinx.serialization.Serializable
+import no.nav.helsearbeidsgiver.utils.wrapper.Fnr
+import java.util.UUID
+
+@Serializable
+@JvmInline
+value class KafkaKey private constructor(
+    internal val key: String,
+) {
+    companion object {
+        operator fun invoke(forespoerselId: UUID): KafkaKey = KafkaKey(forespoerselId.toString())
+
+        operator fun invoke(sykmeldtFnr: Fnr): KafkaKey = KafkaKey(sykmeldtFnr.verdi)
```

Comment:
Snedig. Dette er for å slippe `.toString()` og `.verdi` rundt om kring i koden?

Reply:
Det er mer for å håndheve en streng type. Hvis man har lov til å bruke en hvilken som helst streng som Kafka-nøkkel så blir det ville vesten 🤠